src/components/Footer.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <div class="footer">
+    <!-- WirVsVirus Hackathon Logo -->
+    <div>
+      <div class="logo-wrapper">
+        <img
+          class="wir-vs-virus-logo"
+          src="@/assets/Logo_WirVsVirus.png"
+          :alt="$t('Info.WirVsVirusLogo')"
+        />
+      </div>
+      <p class="logo-caption">
+        {{$t('Info.Created.First')}}
+        <a
+          href="https://wirvsvirushackathon.org/"
+          target="_blank"
+        >{{$t('Info.Created.Link')}}</a>
+      </p>
+    </div>
+    <div class="github-container">
+      <img class="github-logo" src="@/assets/github-mark.png" alt="Github Logo" />
+      <p class="logo-caption">
+        <a href="https://github.com/ProjektKuro/" target="_blank">Github-Repository</a>
+      </p>
+    </div>
+  </div>
+</template>
+
+<style scoped>
+a {
+  color: inherit;
```

Comment:
Links should not have **no** style. 

Please add `border-bottom: 1px solid lightgrey;` like in the navbar.